pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-8196-activity-groups.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="lkapanad" id="activity-groups">
+        <createTable tableName="activity_category">
+            <column name="category_id" type="bigint" autoIncrement="true" startWith="1">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="study_id" type="bigint">
+            </column>
+            <column name="category_name" type="varchar(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="category_code" type="varchar(50)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="category_group">
+            <column name="form_id" type="bigint" autoIncrement="true" startWith="1">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="category_id" type="bigint">
+            </column>
+            <column name="form_code" type="varchar(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="form_name" type="varchar(50)">
```

Comment:
Are we sure that 50 symbols are enough for the form name? I have no idea what names potentially can be used, but something tells me they might be longer than 50 characters.